lib/reports/resolvers.ts

```diff
@@ -1,4 +1,4 @@
-import { createObjectCsvStringifier, createObjectCsvWriter } from 'csv-writer';
+import { createObjectCsvStringifier } from 'csv-writer';
```

Comment:
This is unrelated to the PR changes but required to resolve the CI issue

Reply:
Thanks for this fix!